### PR TITLE
Fidelity op

### PR DESCRIPTION
**Pull request by jaeyoo**
Here, the updated version of fidelity op test is being merged.

### CODE REVIEW

File: tensorflow_quantum/core/ops/math_ops/fidelity_op.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            symbols.
+    Returns:
+        `tf.Tensor` with shape [batch_size, n_others] where `out[i][j]` is equal
+            to the fidelity of `programs[i]` with `symbol_values[i]`
+            resolved in and `other_programs[i][j]`.
+    """
+    ip = inner_product_op.inner_product(programs, symbol_names,
+                                        tf.cast(symbol_values, tf.float32),
+                                        other_programs)
+    return tf.math.abs(ip)**2
```
Comment (MichaelBroughton) [inline tensorflow_quantum/core/ops/math_ops/fidelity_op.py:84]: Needs a newline at the end of file.

File: tensorflow_quantum/core/ops/math_ops/fidelity_op_test.py [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+        with tf.GradientTape() as tape:
+            tape.watch(empty_values)
+            out = fidelity_op.fidelity(empty_circuit, empty_symbols,
+                                       empty_values, other_program)
+
+        self.assertShapeEqual(np.zeros((0, 0)), out)
+
+
+if __name__ == "__main__":
+    tf.test.main()
```
Comment (MichaelBroughton) [inline tensorflow_quantum/core/ops/math_ops/fidelity_op_test.py:305]: Ditto.

**Review by MichaelBroughton — APPROVED** [review]
AMAZING! It felt like it was something simple but I couldn't figure it out late last night. All we need is some newlines at the end of the py modules and we should be good to merge. Thanks so much Jae.

File: tensorflow_quantum/core/ops/math_ops/fidelity_op_test.py (hunk 2 above)
Comment (jaeyoo) [reply tensorflow_quantum/core/ops/math_ops/fidelity_op_test.py]: Done.

**Review by MichaelBroughton — APPROVED** [review]

**Pull request merged**